src/py42/clients/orgs.py

```diff
@@ -1,8 +1,225 @@
 import json
 
 from py42 import settings
+from py42._internal.compat import ChainMap
 from py42.clients import BaseClient
 from py42.clients.util import get_all_pages
+from py42.exceptions import Py42Error
+from py42.settings import debug
+from py42.util import bool_required
+from py42.util import bool_to_str
+from py42.util import str_to_bool
+
+
+class OrgSettingsManager(object):
+    def __init__(self, org_client, org_id):
+        org_response = org_client.get_by_id(org_id, incSettings=True)
```

Comment:
Is there a way to achieve this without making networking calls in the ctor?

Reply:
Yeah, that's how I initially did it (had `sdk.orgs.get_settings_manager()` make the calls then `OrgSettingsManager` just took the two settings dicts as arguments). I thought it might be useful for people to be able to construct the object themselves without having to make custom GET requests, but it's a bit awkward since you still need to pass in the org_client.. Maybe I'll change it back so there's only one way to get OrgSettingsManagers and that's through `.get_settings_manager()`